src/Sylius/Bundle/ApiBundle/Resources/config/api_resources/Adjustment.xml

```diff
@@ -25,20 +25,16 @@
         <collectionOperations />
 
         <itemOperations>
-            <itemOperation name="get">
-                <attribute name="security">is_granted('ROLE_API_ACCESS')</attribute>
+            <itemOperation name="get_adjustments_in_admin">
```

Comment:
```suggestion
            <itemOperation name="get_in_admin">
```

Reply:
I would recommend going with something like this:
```suggestion
            <itemOperation name="sylius_api_admin_adjustments_index">
```
With the following explanation: 
* `sylius_api` as a new prefix
* `admin`/`shop` section prefix
* `resources` resource name (from Sylius resource)
* `operation` one of default (index, show, create, update, delete) or custom if needed. 

This proposal would be pretty similar to what we have already in Sylius route generation. Drawback of it is that all routes should be named right now.

Reply:
Or improve the currently generated paths 
![image](https://user-images.githubusercontent.com/6213903/92392562-be2aa480-f11e-11ea-854a-f42aee9787f3.png)
with adding `admin` or `shop` prefix at the beginning

Reply:
But the operation name that we can define here is used by API Platform to generate route name
Example for `<itemOperation name="add_item">` for Order resource: ![image](https://user-images.githubusercontent.com/6140884/92440720-3a6bc900-f1ad-11ea-9422-74e0e651f666.png)
So I would be for sth more like `admin_get` or `get_in_admin`